windows_install_service.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+
+setlocal
+
+set SERVICE=cloud-sql-proxy
+set DISPLAYNAME=Google Cloud SQL Auth Proxy
+set CREDENTIALSFILE=%~dp0key.json
+set CONNECTIONNAME=project-id:region:db-instance
```

Comment:
This could just be `CSQL_INSTANCE_CONNECTION_NAME` and doesn't need to be set in the CLI invocation (unless that seems more clear).

See https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/main/cmd/root.go#L239.

Reply:
See answer above: https://github.com/GoogleCloudPlatform/cloud-sql-proxy/pull/1696#discussion_r1149711710